tidy3d/plugins/mode/mode_solver.py

```diff
@@ -173,6 +177,11 @@ def plane_in_sim_bounds(cls, val, values):
             raise SetupError("'ModeSolver.plane' must intersect 'ModeSolver.simulation'.")
         return val
 
+    def _post_init_validators(self) -> None:
+        validate_mode_plane_radius(
```

Comment:
not sure if we put docstrings for these in other places, but a short one wouldnt hurt (validation check that runs after object initialized).